src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts

```diff
@@ -1438,6 +1438,41 @@ export const whereDoChildrenLive = (
   };
 };
 
+export const otherPersonConfidentiality = (
+  { sectionTitles, keys, ...content }: SummaryListContent,
+  userCase: Partial<CaseWithId>,
+  language
+): SummaryList | undefined => {
+  const sessionOtherPeopleData = userCase['oprs_otherPersons'];
+  const newOtherPeopleStorage: {
+    key: string;
+    keyHtml?: string;
+    value?: string;
+    valueHtml?: string;
+    changeUrl: string;
+  }[] = [];
+
+  for (const otherPerson in sessionOtherPeopleData) {
+    const firstName = sessionOtherPeopleData[otherPerson]['firstName'],
+      lastName = sessionOtherPeopleData[otherPerson]['lastName'],
+      id = sessionOtherPeopleData[otherPerson]['id'];
+    const isOtherPersonAddressConfidential = sessionOtherPeopleData[otherPerson]?.['isOtherPersonAddressConfidential'];
+
+    if (isOtherPersonAddressConfidential !== undefined) {
```

Comment:
if(isOtherPersonAddressConfidential) should do

Reply:
Removed when adding the inline error message.